example/examplechecker.py

```diff
@@ -1,128 +1,190 @@
                     self.flag_idx
                 )
             )
 
-    def getflag(self):  # type: () -> None
+    def getflag(self) -> None:
         """
         This method retrieves a flag from the service.
         Use self.flag to get the flag that needs to be recovered and self.round to get the round the flag was placed in.
         On error, raise an EnoException.
         :raises EnoException on error
-        :return this function can return a result if it wants
-                if nothing is returned, the service status is considered okay.
-                the preferred way to report errors in the service is by raising an appropriate enoexception
         """
         if self.flag_idx == 0:
-            if not self.team_db.get(sha256ify(self.flag), None) == self.flag:
-                raise BrokenServiceException("We did not get flag 0 back :/")
+            credentials = self.team_db[self.flag]
+            self.login(credentials)
+
+            res = self.http_get("/notes")
+            assert_equals(res.status_code, 200)
+
+            try:
+                if self.flag not in res.json()["notes"]:
+                    raise BrokenServiceException("flag is missing from /notes")
+            except (KeyError, json.JSONDecodeError):
+                raise BrokenServiceException(
+                    "received invalid response on /notes endpoint"
+                )
+
         elif self.flag_idx == 1:
-            if (
-                not self.global_db.get("{}_{}".format(self.address, self.flag), None)
-                == "Different place for "
-                "different flag_idx"
-            ):
-                raise BrokenServiceException("Flag 2 was missing. Service is broken.")
+            credentials = self.team_db[self.flag]
+            self.login(credentials)
+
+            res = self.http_get("/profile")
+            assert_equals(res.status_code, 200)
+
+            try:
+                if self.flag != res.json()["status"]:
+                    raise BrokenServiceException("flag is missing from /profile")
+            except (KeyError, json.JSONDecodeError):
+                raise BrokenServiceException(
+                    "received invalid response on /profile endpoint"
+                )
         else:
             raise ValueError(
                 "Call_idx {} not supported!".format(self.flag_idx)
             )  # Internal error.
 
-    def putnoise(self):  # type: () -> None
+    def putnoise(self) -> None:
         """
         This method stores noise in the service. The noise should later be recoverable.
         The difference between noise and flag is, tht noise does not have to remain secret for other teams.
         This method can be called many times per round. Check how often using self.flag_idx.
         On error, raise an EnoException.
         :raises EnoException on error
-        :return this function can return a result if it wants
```

Comment:
Same question as above

Reply:
> Minor comments

I accidentally included the commits from #53 (which you are refering to) here, I will work on the fixes in #53 instead